front/src/utils/date.ts

```diff
@@ -177,6 +177,13 @@ export function convertIsoUtcToLocalTime(isoUtcString: string): string {
   return dayjs(isoUtcString).local().format();
 }
 
+export function convertToLocalTime(isoString: string): string {
```

Comment:
I'd prefer not to add this function to `date.ts`: it forces UTC without converting. It's not something that one should do in general (although here it's necessary to workaround a space time chart bug).

Reply:
You're right, there was a much simpler way to fix this bug